crates/pevm/src/storage/rpc.rs

```diff
@@ -51,36 +57,47 @@ impl<N: Network> RpcStorage<N> {
             cache_accounts: Mutex::default(),
             cache_bytecodes: Mutex::default(),
             cache_block_hashes: Mutex::default(),
-            // TODO: Better error handling.
-            runtime: Runtime::new().unwrap(),
+            runtime: OnceLock::new(),
+        }
+    }
+
+    #[inline(always)]
```

Comment:
Really like the implementation now! But to simplify we can remove this (we don't need to micro-optimize this module) and the comments as the code is pretty self-explanatory. 

If we were to add a comment I'd add that we currently only support a global multi-thread runtime.